spec/factories/users.rb

```diff
@@ -98,7 +98,7 @@
 
     trait :wct_china_member do
       after(:create) do |user, options|
-        FactoryBot.create(:team_member, team_id: Team.wct_china.id, user_id: user.id, team_senior_member: options.team_senior_member, team_leader: options.team_leader)
```

Comment:
You're losing two important bits of information (senior member, team leader). For WCT China, this is probably okay because they don't have so strict structures, but what about future role migrations?

Reply:
For WCT China, the status is supposed to be `member` in testcases used so it's fine. For future migration, I'm planning to use multiple traits like `wrt_member`, `wrt_senior_member`, etc.